### PR TITLE
initialize new role

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: roles/pulibrary.timezone/molecule/default/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    timezone: America/New_York
+  roles:
+    - role: pulibrary.timezone
```
Comment (christinach) [inline roles/pulibrary.timezone/molecule/default/playbook.yml:7]: Can we add a var for the timezone in this PR? Or does this needs to be done in a separate PR?

Comment (kayiwa) [reply roles/pulibrary.timezone/molecule/default/playbook.yml:7]: there's one in [defaults/main.yml](defaults/main.yml). The benefit here is if one adds the variable `timezone: America/New_York` in their `group_vars/yourrole/vars.yml` it will add that.
